src/series/learn-git.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: layouts/series/index.njk
+title: Learn Git
```

Comment:
Just a thought — I wonder if we called these series' something like `Git Fundamentals` or `Git Basics`. Take or leave the idea!